### ISSUE TITLE
XML sitemap is not returned properly

### ISSUE THREAD

**Issue by Sexycrets** [waterlink/rack-reverse-proxy#17]
Hi,

Everything works perfectly BUT the sitemap is not returned properly.

Although I am sure the sitemap.xml is where it is supposed to be, you can see it loads properly with direct URL:
http://blog.sexycrets.com/sitemap.xml

When I try to get it using the reverse proxy (which works perfectly for http://www.sexycrets.com/blog/):
http://www.sexycrets.com/blog/sitemap.xml
the XML is not returned.

What should I do? The only reason why I am using the reverse proxy is for SEO purposes but if the sitemap is not returned that defeats the whole purpose.

Thanks!

**Comment by Sexycrets** [comment]
Turns out that the XML is returned as expected from the reverse proxy. The reason why on the browser you see it differently is (I guess) because Wordpress senses the user-agent being a browser and returns a well formatted human readable html with the XML content within.

I tried to submit the sitemap to google for test through the reverse proxy URL and it reads it OK. So I confirm this reverse proxy gem works perfectly with XMLs too.
